setup.py

```diff
@@ -31,6 +31,10 @@
             "transformers==4.2.2",
             "protobuf==3.19.4",
         ],
+        "test_scispacy": [
```

Comment:
I believe these dependencies are needed when using the newly created processor. So we shouldn't name this `test_scispacy`. In addition, we should have some documentation (such as in Readme.md), to talk about how to setup the environment for certain processors if you want to use it.